components/service/sync-logins/README.md

```diff
@@ -161,6 +161,12 @@ Currently there is no way to change the key, once set (see https://github.com/mo
 
 Part is in this tree, but most is in https://github.com/mozilla/application-services. Specifically, there are two pieces not present in this repository, an [android-specific piece written in Kotlin](https://github.com/mozilla/application-services/tree/master/logins-sql/tree/master/logins-api/android), and a [cross-platform piece written in Rust](https://github.com/mozilla/application-services/tree/master/logins-sql).
 
+### What's the difference between `reset`, `wipe` and `wipeLocal`?
+
+- `reset` deletes last sync timestamps, and what the database believes is stored remotely. This is almost never what you want, and so it is deprecated.
```

Comment:
Do we have any consumers even using it `reset`? I'd just remove it if not...

Reply:
Hm, well, yes, lockbox uses it. But they should use `wipeLocal` instead. I don't mind taking it out here.